app/addons/databases/components.js

```diff
@@ -179,12 +283,17 @@ class DatabaseRow extends React.Component {
         </tr>
       );
     }
+
     const partitionedCol = this.props.showPartitionedColumn ?
       (<td>{isPartitioned ? 'Yes' : 'No'}</td>) :
       null;
+
+    const DEFAULT_PADDING = 8;
```

Comment:
I felt a little bad copying this from the css, any thoughts on a nicer way to handle the indenting?